kolibri/plugins/facility/assets/src/views/DataPage/SyncInterface/index.vue

```diff
@@ -126,6 +117,7 @@
     SyncFacilityModalGroup,
   } from 'kolibri.coreVue.componentSets.sync';
   import commonSyncElements from 'kolibri.coreVue.mixins.commonSyncElements';
+  import responsiveWindowMixin from 'kolibri.coreVue.mixins.responsiveWindowMixin';
```

Comment:
This is non-blocking but just flagging that we now have the `useKResponsiveWindow` composable that provides for better code readability and control. Here is a [sample usage](https://github.com/learningequality/kolibri/blob/4ad3e289ddf4a00d91f5bccfdf7977147611d0f9/kolibri/plugins/learn/assets/src/views/LibraryPage/SidePanel.vue#L77) of the same

Reply:
Thanks for pointing this out and providing an example usage. I have noticed some areas where more control would be beneficial, particularly for the Kolibri app, so I will update this PR to use `useKResponsiveWindow` instead.